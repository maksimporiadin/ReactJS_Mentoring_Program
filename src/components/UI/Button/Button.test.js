import 'jsdom-global/register';
import React from 'react';

import { Button } from './index';

import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

configure({adapter: new Adapter()});

describe('Button', () => {
    let wrapper;
    const defaultProps = {
        className: 'title',
        disabled: true,
        onClick() {},
        name: 'name'
    };

    beforeEach(()=>{
        jest.resetAllMocks();
        wrapper = shallow(<Button {...defaultProps} />)
    });

    it('is renders', () => {
        expect(wrapper).toMatchSnapshot();
    });
});