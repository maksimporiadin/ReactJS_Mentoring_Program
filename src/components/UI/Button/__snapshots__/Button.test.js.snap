// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button is renders 1`] = `
<button
  className="btn button--default title"
  disabled={true}
  onClick={[Function]}
>
  name
</button>
`;
